root/static/scripts/edit/URLCleanup.js

```diff
@@ -2792,6 +2792,24 @@ const CLEANUPS = {
               result: prefix === 'artist',
               target: ERROR_TARGETS.RELATIONSHIP,
             };
+          case LINK_TYPES.lyrics.release_group:
+            if (prefix === 'album') {
+              return {
+                error: exp.l(
+                  `Musixmatch “{album_url_pattern}” pages are a bad match
+                   for MusicBrainz release groups, and linking to them
+                   is currently disallowed.`,
+                  {
+                    album_url_pattern: (
+                      <span className="url-quote">{'/album'}</span>
+                    ),
+                  },
+                ),
+                target: ERROR_TARGETS.RELATIONSHIP,
```

Comment:
It seems no other relationship type would fit and this album URL is just a bad match for this entity type.

```suggestion
                target: ERROR_TARGETS.ENTITY,
```

No other entity type would fit either but the relationship type selector can be eventually hidden at least.